tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = py39
```

Comment:
Inconsistent with `setup.cfg` where 3.8 is the minimum. CI should use the lowest supported version. I think switching to 3.9 should be ok though.